src/main/php/CommandLine/FileFinders/FileFinderInterface.php

```diff
@@ -1,7 +1,20 @@
 <?php
+
 namespace Zooroyal\CodingStandard\CommandLine\FileFinders;
 
+use Zooroyal\CodingStandard\CommandLine\ValueObjects\GitChangeSet;
+
 interface FileFinderInterface
 {
-    public function findFiles($filter = '', $stopword = '', $targetBranch = '');
+    /**
+     * This method searches for files by using Git as indexing service.
+     *
+     * @param string $filter
+     * @param string $blacklistToken
+     * @param string $whitelistToken
+     * @param string $targetBranch
+     *
+     * @return GitChangeSet
+     */
+    public function findFiles($filter = '', $blacklistToken = '', $whitelistToken = '', $targetBranch = ''): GitChangeSet;
```

Comment:
string typehints?

Reply:
jo, aber dann auch noch alle typehints in implementierungen nachziehen ^^